tests/unit/test_world_scientific.py

```diff
@@ -67,158 +59,358 @@ def override_generated_fields(record):
     return record
 
 
-def test_abstract(results):
+@pytest.mark.parametrize(
+    'generated_abstract, expected_abstract',
+    [
+        [
+            get_results('world_scientific/sample_ws_record.xml'),
+            "CH$_{3}$NH$_{3}$PbX(X = Br, I, Cl) perovskites have recently been used as light absorbers in hybrid"
+            " organic-inorganic solid-state solar cells, with efficiencies above 15%. To date, it is essential to"
+            " add Lithium bis(Trifluoromethanesulfonyl)Imide (LiTFSI) to the hole transport materials (HTM) to get"
+            " a higher conductivity. However, the detrimental effect of high LiTFSI concentration on the charge transport"
+            ", DOS in the conduction band of the TiO$_{2}$ substrate and device stability results in an overall "
+            "compromise for a satisfactory device. Using a higher mobility hole conductor to avoid lithium salt "
+            "is an interesting alternative. Herein, we successfully made an efficient perovskite solar cell by "
+            "applying a hole conductor PTAA (Poly[bis(4-phenyl) (2,4,6-trimethylphenyl)-amine]) in the absence of"
+            " LiTFSI. Under AM 1.5 illumination of 100 mW/cm$^{2}$, an efficiency of 10.9% was achieved, which is "
+            "comparable to the efficiency of 12.3% with the addition of 1.3 mM LiTFSI. An unsealed device without "
+            "Li$^{+}$ shows interestingly a promising stability.",
+        ],
+    ],
+    ids=[
+        'test wsp abstract',
```

Comment:
I'd use just "simple" ass 'test_abstract' is already in the main test name (will be shown as: "<main_test_name>[<id>]")